cmd/authority-claimer/build.rs

```diff
@@ -12,7 +12,7 @@ use std::{
 };
 
 const ROLLUPS_CONTRACTS_URL: &str =
-    "https://registry.npmjs.org/@cartesi/rollups/-/rollups-2.0.0-rc.2.tgz";
+    "https://registry.npmjs.org/@cartesi/rollups/-/rollups-2.0.0-rc.5.tgz";
```

Comment:
Just a reminder, we plant to amend or replace 36e13a84 with the latest contracts RC
We should also update `.gitmodules` pointing to the right tag.